index.html

```diff
@@ -174,5 +174,43 @@ <h3 class="card-title">TONIC</h3>
       </ul>
     </div>
   </section>
+  <footer class="contact-section">
+    <div class="contact">
+      <h2>Contact me</h2>
+      <p class="contact-section-text-content">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="https://formspree.io/f/mgerkggg" method="POST">
+        <input
+          type="text"
+          name="user-name"
+          id="user-name"
+          required
+          placeholder="Name"
+          maxlength="30"
+        />
+        <input
+          type="email"
+          name="user-email"
+          id="user-email"
+          required
+          placeholder="Email"
+        />
+        <textarea
+          name="message"
+          id="message"
+          required
+          placeholder="Write your message here"
+          maxlength="500"
+        >
+        </textarea>
```

Comment:
- [ ] You need to stick to the design templates in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding) 
- This section is missing the necessary background image.
- You are also missing the placeholder for the message.
- See both images below and fix the differences, take note of the font used in the placeholders.

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/122085199-56913c00-cdb7-11eb-9aa9-1126217f4a1a.png) | ![image](https://user-images.githubusercontent.com/49621279/122084746-f5696880-cdb6-11eb-9564-34cbb469d410.png)

Reply:
Fixed required changes, thank you.

Reply:
done 👍